apps/emqx_prometheus/src/emqx_prometheus.erl

```diff
@@ -310,6 +313,7 @@ aggre_or_zip_init_acc() ->
         emqx_olp_data => meta_to_init_from(olp_metric_meta()),
         emqx_acl_data => meta_to_init_from(acl_metric_meta()),
         emqx_authn_data => meta_to_init_from(authn_metric_meta()),
+        cluster_rpc => meta_to_init_from(cluster_rpc_meta()),
```

Comment:
It's a bit hard to follow the path for those prometheus callbacks, but it seems like putting this here might lead to the transaction id being summed over the cluster<sup>[[1]](https://github.com/emqx/emqx/blob/22e595fa96fdaf75fc9a02ae9e6146a1a1b31bb6/apps/emqx_prometheus/src/emqx_prometheus_cluster.erl#L81)</sup><sup>[[2]](https://github.com/emqx/emqx/blob/22e595fa96fdaf75fc9a02ae9e6146a1a1b31bb6/apps/emqx_prometheus/src/emqx_prometheus_cluster.erl#L116)</sup>?

Reply:
Honestly, it doesn't add up.
This is because when it comes to getting aggregated metrics, its raw data carries the node as Label, e.g:
```
curl -X 'GET' \
  'http://127.0.0.1:18083/api/v5/prometheus/stats?mode=all_nodes_aggregated' \
  -H 'accept: text/plain' | grep emqx_conf

# TYPE emqx_conf_sync_txid gauge
# HELP emqx_conf_sync_txid
emqx_conf_sync_txid{node="emqx@127.0.0.1"} 6
```

for the node metrics, it looks like:
```
curl -X 'GET' \
  'http://127.0.0.1:18083/api/v5/prometheus/stats?mode=node' \
  -H 'accept: text/plain' | grep emqx_conf

# TYPE emqx_conf_sync_txid gauge
# HELP emqx_conf_sync_txid
emqx_conf_sync_txid 6
```